docs/source/kedro_project_setup/dependencies.md

```diff
@@ -59,7 +36,36 @@ This installs Kedro and dependencies related to the data type group. An example
 To limit installation to dependencies specific to a data type:
 
 ```bash
-pip install "kedro-datasets[<group>.<dataset>]"
+pip install "kedro-datasets[<group>-<dataset>]"
```

Comment:
I wonder is it worth to have annotation like "Changed in kedro-datasets 3.0", similar to this in the Python docs.
https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods

Reply:
I haven't changed it everywhere the dataset dependencies have been mentioned, just updated this page, but something we could do in https://github.com/kedro-org/kedro/pull/3664? (cc @lrcouto )